cluster/examples/kubernetes/ceph/operator-with-csi.yaml

```diff
@@ -531,9 +525,3 @@ spec:
         emptyDir: {}
       - name: default-config-dir
         emptyDir: {}
-      - name: csi-rbd-config
```

Comment:
don't we need to update document for this one? https://github.com/rook/rook/blob/master/Documentation/ceph-csi-drivers.md#create-csi-driver-deployment-templates-and-persist-them-in-configmaps

Reply:
Yep. I noted that I still need to update the docs in the description.

Reply:
@Madhu-1 I will update the documentation once the discussion around what to do wrt the templates (in pod or not) is resolved.